test Fixture with speci@l chars/.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
     "java.project.referencedLibraries": [
         "lib/**/*.jar",
         "/home/apupier/.config/Code/User/globalStorage/redhat.vscode-camelk/java-dependencies-3.0.0/*.jar"
```

Comment:
seems to be some local setup? do we need this in master branch?

Reply:
Effectively a local setup and seems no need but not modified in this PR.
Planning to remove it in another PR.
There is also the didact.registered.
Maybe th ewhoel file shoudl be removed. I need to check when master will be back to green.

Reply:
see https://github.com/camel-tooling/camel-lsp-client-vscode/pull/688